fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -4080,7 +4084,7 @@ struct StartFullRestoreTaskFunc : RestoreTaskFuncBase {
 		for (auto const& r : ranges) {
 			keyRangesFilter.push_back_deep(keyRangesFilter.arena(), KeyRangeRef(r));
 		}
-		Optional<RestorableFileSet> restorable =
+		state Optional<RestorableFileSet> restorable =
```

Comment:
Nit: You don't need `state` if the `wait(tr->commit)` happens after the last use of `restorable`.

Reply:
I'd prefer committing the transaction of setting firstConsistentVersion separately for better readability.